core/domain/android_validation_constants.py

```diff
@@ -23,14 +23,15 @@
 # changed, just this array has to be modified to include the new
 # interactions or remove existing ones.
 VALID_INTERACTION_IDS = [
-    'Continue', 'EndExploration', 'NumericInput', 'TextInput',
-    'FractionInput', 'NumberWithUnits', 'MultipleChoiceInput',
-    'ItemSelectionInput']
+    'Continue', 'DragAndDropSortInput', 'EndExploration', 'FractionInput',
+    'ItemSelectionInput', 'MultipleChoiceInput', 'NumericInput',
+    'NumberWithUnits', 'TextInput'
+    ]
 
 SUPPORTED_LANGUAGES = ['en']
 
 INVALID_RTE_COMPONENTS = [
-    'collapsible', 'math', 'skillreview', 'tabs', 'video']
+    'collapsible', 'math', 'tabs', 'video']
```

Comment:
Should this instead be a VALID_RTE_COMPONENTS whitelist? Otherwise, when a new RTE component is added, it would automatically be included.

Reply:
Yeah, makes sense. Done.